app/src/main/java/com/example/immadisairaj/quiz/QuizActivity.java

```diff
@@ -272,6 +272,9 @@ public void onClick(DialogInterface dialog, int id) {
         });
         AlertDialog scoreDialog = builder.create();
         scoreDialog.show();
+        Button button = findViewById(R.id.submit);
+        button.setVisibility(View.INVISIBLE);
```

Comment:
If it becomes invisible. I press outside the dialogue box and there is no further way for me to go.